record-editor.py

```diff
@@ -361,6 +358,12 @@ def create_widgets( self ):
         self.infoTakenLabel    = QLabel()
         self.infoTagsLabel     = QLabel()
 
+        # dock widget which will hold the selection layout once created
+        # in create_layout, for now it gets an empty widget.
+        self.selection_dock = QDockWidget()
```

Comment:
The idea was to set the layout stretch factor(s) such that the scaling image portion of the window gets all extra space to expand (or contract) from resizing of the window, yet still have a way to easily resize the selection view if necessary. You also just get some niceties for free with dock widgets, like being able to reposition dock widgets to any of the available dock areas (top, bottom, left, right), or have them float outside the main window (though I disabled this since I didn't think it necessary). Note that this was not intended to solve the problem of the selection view not being fully visible with regards to is column sizes at initialization. I think that will take creating a subclass of whatever View class to be able to override sizeHint().